.coveragerc

```diff
@@ -0,0 +1,7 @@
+[run]
+branch = True
+source = AccessControl
+
+[report]
+precision = 2
+omit = */tests.py
```

Comment:
Can you add `show_missing = True` to the `[report]` section, too?  We are missing two lines at present.